src/views/CardStack/CardStack.js

```diff
@@ -102,6 +103,9 @@ const animatedSubscribeValue = (animatedValue: Animated.Value) => {
 };
 
 class CardStack extends React.Component<Props> {
+  static defaultProps = {
+    globalGesturesEnabled: Platform.OS === 'ios',
```

Comment:
💁 If the prop isn't used, platform will be used to determine whether to enable globalGestures. This also means that if we use this prop, we will be responsible for disabling gestures on Android.